README.md

```diff
@@ -23,13 +23,25 @@ gen-mcp eliminates the complexity of building Model Context Protocol (MCP) serve
 - **📡 OpenAPI Auto-Conversion** - Transform existing OpenAPI specs into MCP servers instantly
 - **🔄 Real-Time Tool Exposure** - HTTP endpoints become callable AI tools automatically
 - **🛡️ Built-in Validation** - Schema validation and type safety out of the box
+- **🔐 Security Out of the Box** - TLS encryption and OAuth/OIDC authentication built-in
 - **⚡ Background Processing** - Detached server mode with process management
 - **🔧 Flexible Configuration** - Fine-tune which endpoints to expose and how they behave
 
 ## 🚀 Quick Start
 
-### 1. Build the Binaries
+### 1. Install gen-mcp
 
+**Option A: Download Pre-built Binary**
+```bash
+# Download from GitHub releases
+# Visit: https://github.com/genmcp/gen-mcp/releases
+# Or using curl (replace with latest version):
+curl -L https://github.com/genmcp/gen-mcp/releases/latest/download/genmcp-linux-amd64 -o genmcp
```

Comment:
maybe we can name the release files like `genmcp-Linux-x86_64`, so that users could download via something like  `https://github.com/genmcp/gen-mcp/releases/latest/download/genmcp-$(uname -s)-$(uname -m) -o genmcp`

Reply:
I like that idea - want to open an issue?